exercises/wordy/example.rs

```diff
@@ -0,0 +1,82 @@
+#[derive(Debug, PartialEq)]
+pub struct WordProblem {
+    command: String,
+}
+
+#[derive(Debug, PartialEq)]
+struct Token {
+    value: String,
+}
+
+impl Token {
+    fn is_valid(&self) -> bool {
+        !self.value.is_empty() && (self.is_operand() || self.is_operator())
+    }
+
+    fn is_operand(&self) -> bool {
+        self.value.chars().all(|c| c.is_numeric() || c == '-')
+    }
+
+    fn is_operator(&self) -> bool {
+        self.value == String::from("plus") || self.value == String::from("minus") ||
+        self.value == String::from("multiplied") || self.value == String::from("divided")
+    }
+}
+
+impl WordProblem {
+    pub fn new(c: &str) -> Self {
+        WordProblem { command: String::from(c) }
+    }
+
+    pub fn answer(&self) -> Result<isize, ()> {
+        let mut t = self.tokens();
+        let mut result: isize = 0;
+        let mut opr = "plus".to_string();
+
+        if t.len() <= 1 {
+            Err(())
```

Comment:
stylistically I may have considered making this a `return`, then unindenting the rest (removing the `else`), but not something I'd urgently change